README.md

```diff
@@ -38,7 +38,7 @@ The Microsoft Identity Web roadmap is available from [Roadmap](https://github.co
 - Code samples are available for [web app samples](https://github.com/AzureAD/microsoft-identity-web/wiki/web-app-samples)
   and [web API samples](https://github.com/AzureAD/microsoft-identity-web/wiki#web-api-samples)
 
-## Where do I file issues
+## Where do I file issues?

```

Comment:
nit: having a question mark in the section heading may be an issue if someone links to this heading. I vaguely recall issues with links like this previously.

Reply:
fixed